toolchain/driver/compile_subcommand.cpp

```diff
@@ -294,6 +295,35 @@ Whether to emit DWARF debug information.
         arg_b.Default(true);
         arg_b.Set(&include_debug_info);
       });
+  b.AddFlag(
+      {
+          .name = "poison_verbose",
+          .help = R"""(
+Print verbose logs of ASAN poisoning events in the value stores. Used to debug
+ASAN use-after-poison crashes, and only has an effect when run with the ASAN
+sanitizer.
+)""",
+      },
+      [&](auto& arg_b) {
+        arg_b.Default(false);
+        arg_b.Set(&poison_verbose);
+      });
+  b.AddStringOption(
+      {
+          .name = "poison_stop",
```

Comment:
To me, "--poison-stop" sounds a lot like it simply stops poisoning. From the description, maybe `--poison-abort-at`?

```suggestion
          .name = "poison-abort-at",
```

Reply:
Sure, can do

Reply:
I shortened this slightly for easier typing, with `--poison_abort`, hopefully that sounds okay too?